README.md

```diff
@@ -30,7 +30,7 @@ import com.lorandszakacs.enclosure.Enclosure
 
 object Printer {
   def locatedPrintln(s: String)(implicit enc: Enclosure): Unit =  {
-    println(s"[${enc.fullModuleName}] s")
+    println(s"[${enc.fullModuleName}] $s")
```

Comment:
this breaks scaladoc. need to find out how to insert literal '$' char :grimacing:

Reply:
`Compile / doc` in sbt to see why.

Reply:
`$$` seems to fix this, I've changed it in `Enclosure.scala` and sbt seems to embrace this 😅

Reply:
great. Thank you! :+1: